src/generic_ui/polymer/cloud-install.ts

```diff
@@ -7,28 +7,36 @@ import ui_constants = require('../../interfaces/ui');
 
 var ui = ui_context.ui;
 
-const INSTALL_ARGS: uproxy_core_api.CloudInstallArgs = {
-  providerName: 'digitalocean'
+const DEFAULT_REGION = 'nyc3'
+
+var install_args: uproxy_core_api.CloudInstallArgs = {
```

Comment:
Ah, got it. I reverted those changes and deleted the install_args variable from cloud-install.ts.